pkg/webhook/pod/tikv_deleter.go

```diff
@@ -100,9 +100,12 @@ func (pc *PodAdmissionControl) admitDeleteTiKVPods(payload *admitPayload) *admis
 	case v1alpha1.TiKVStateTombstone:
 		return pc.admitDeleteUselessTiKVPod(payload)
 	case v1alpha1.TiKVStateOffline:
-		return pc.admitDeleteOfflineTiKVPod()
+		return pc.rejectDeleteTiKVPod()
 	case v1alpha1.TiKVStateDown:
-		return pc.admitDeleteUselessTiKVPod(payload)
+		// When the target tikv's store is DOWN, we would not pass the deleting request, otherwise it would cause
+		// the duplicated id problem for the newly tikv pod.
+		// Users should offline the target tikv into tombstone first, then scale-in it.
```

Comment:
The logic in tikv scaler is to delete the store automatically https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/tikv_scaler.go#L110, now we request users to delete the store manually, what do you think? @cofyc @weekface